pkg/logs/sender/sender.go

```diff
@@ -67,7 +76,7 @@ func (s *Sender) send(payload *message.Message) {
 				break
 			}
 			switch err.(type) {
-			case *client.FramingError:
+			case *tcp.FramingError:
```

Comment:
`sender` should be agnostic from the inner transport so we should not have a reference to the `tcp` package.